frontend/src/workStation/WorkStation.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Nav from "../app/Nav";
+import Footer from "../app/Footer";
+import CodeIDE from "./CodeIDE";
+import { useParams } from "react-router-dom";
+import { Box, Text } from "@chakra-ui/react";
+
+function WorkStation() {
+  const [groupData, setGroupData] = useState([]);
+  const [fileData, setFileData] = useState([]);
+  const [error, setError] = useState("");
+  const { groupId, fileId } = useParams();
+
+  const BACKEND_URL = import.meta.env.VITE_BACKEND_URL;
+  const token = localStorage.getItem("jwt");
+
+  const fetchGroupDetails = async () => {
+    try {
+      const response = await axios.get(`${BACKEND_URL}/api/group/${groupId}`, {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+        withCredentials: true,
+      });
+      setGroupData("");
+      setGroupData(response.data);
```

Comment:
Why do you null out `groupData` right before setting it to the response data?

Reply:
This was a mistake. I usually do this for state variables with an array data type to ensure that the old value is overridden; not for string values since it is already implied that the old state will be overridden.